packages/components/src/components/select/select.ts

```diff
@@ -980,131 +984,144 @@ export default class SdSelect extends SolidElement implements SolidFormControl {
               popup:popup__content,
             "
           >
-            <div
-              part="combobox"
-              class=${cx(
-                'relative w-full px-4 flex flex-row items-center rounded-default',
-                this.open && 'shadow',
-                {
-                  sm: 'py-1 min-h-[32px]',
-                  md: 'py-1 min-h-[40px]',
-                  lg: 'py-2 min-h-[48px]'
-                }[this.size]
-              )}
-              slot="anchor"
-              @keydown=${this.handleComboboxKeyDown}
-              @mousedown=${this.handleComboboxMouseDown}
-              @mouseenter=${this.handleMouseEnter}
-              @mouseleave=${this.handleMouseLeave}
-            >
-              <input
-                name=${this.name}
-                form=${this.form}
-                part="display-input"
+            <div class="relative w-full h-full grid grid-cols-1" slot="anchor">
```

Comment:
Wait. So we can now use different triggers as well? Is this useful?

Reply:
No, that is not the case, we already had it setup similar to this before